monitorcontroller/monitorcontroller.go

```diff
@@ -19,8 +19,10 @@ package monitorcontroller
 
 import (
 	"context"
+	"errors"
 
 	"github.com/aoscloud/aos_common/api/cloudprotocol"
+	"github.com/aoscloud/aos_communicationmanager/amqphandler"
```

Comment:
split imports aos_common  and a0s_communicationmanager

Reply:
done